src/module.ts

```diff
@@ -100,8 +100,8 @@ export const plugin = new PanelPlugin<PsyOptions>(PsyPanel).setPanelOptions((bui
     })
     .addMultiSelect({
       path: 'regions',
-      name: 'ASHRAE Comfort Regions',
```

Comment:
All the regions shown are developed by ASHRAE, but I think this is fine to remove, in case we include other standards.